libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -280,6 +280,13 @@ double WaveTrack::Interval::GetPlayEndTime() const
    return mpClip->GetPlayEndTime();
 }
 
+bool WaveTrack::Interval::IntersectsPlayRegion(double t0, double t1) const
+{
+   // TODO wide wave tracks:  assuming that all 'narrow' clips share common
+   // boundaries
```

Comment:
You don't need the comment above.  We have done the work to guarantee it now that this is always so: upgrading the opening of old files with misalignment, and changing what happens when mono tracks are joined into stereo.

Reply:
That doesn't invalidate this statement that we are assuming equal boundaries, though, does it?

Reply:
We do assume that, and there will be nothing left TODO later